tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Azure/Functions/AzureFunctionsCommon.cs

```diff
@@ -21,7 +27,7 @@ internal static class AzureFunctionsCommon
     {
         public const string IntegrationName = nameof(Configuration.IntegrationId.AzureFunctions);
 
-        public const string OperationName = "azure_functions.invoke";
+        public const string OperationName = "azure.functions.invoke";
```

Comment:
I don't think we should change this.
1. It's a breaking change: trace stats use operation names, so any metrics, dashboards, monitors, etc relying on this name will stop working.
2. The general convention for operation names is something like `<product>.<action>` (single dot).

examples (ignore the `-` vs `_`, that's a separate issue 😅)
```
aspnet.request
aspnet-mvc.request
aspnet-webapi.request
aspnet_core.request
aspnet_core_mvc.request
```

(There are many operation names that don't follow this convention. Mistakes were made that are now hard to fix because of point 1 above)